content-script.js

```diff
@@ -0,0 +1,482 @@
+                        </div>
+                        <div class="form-group">
+                            <label for="draft-body">Description</label>
+                            <textarea id="draft-body" class="form-control" rows="10" placeholder="Issue description">${body}</textarea>
+                        </div>
+                    </div>
+                    <div class="draft-modal-footer">
+                        <button class="btn btn-secondary draft-modal-cancel">Cancel</button>
+                        <button class="btn btn-primary draft-modal-save">${isEdit ? 'Update' : 'Save'} Draft</button>
+                        ${isEdit ? '<button class="btn btn-danger draft-modal-delete">Delete Draft</button>' : ''}
+                    </div>
+                </div>
+            `;
+            
+            document.body.appendChild(modal);
+            
+            // Event listeners
+            modal.querySelector('.draft-modal-close').addEventListener('click', () => modal.remove());
+            modal.querySelector('.draft-modal-cancel').addEventListener('click', () => modal.remove());
+            modal.querySelector('.draft-modal-save').addEventListener('click', () => this.saveDraft(modal, existingDraft));
+            
+            if (isEdit) {
+                modal.querySelector('.draft-modal-delete').addEventListener('click', () => this.deleteDraft(modal, existingDraft));
+            }
+            
+            // Close modal when clicking overlay
+            modal.addEventListener('click', (e) => {
+                if (e.target === modal) {
+                    modal.remove();
+                }
+            });
+            
+            // Focus on title input
+            modal.querySelector('#draft-title').focus();
+        },
+        
+        // Save draft from modal
+        async saveDraft(modal, existingDraft) {
+            const titleInput = modal.querySelector('#draft-title');
+            const bodyInput = modal.querySelector('#draft-body');
+            
+            const title = titleInput.value.trim();
+            const body = bodyInput.value.trim();
+            
+            if (!title) {
+                alert('Please enter a title for the draft issue.');
+                titleInput.focus();
+                return;
+            }
+            
+            const draftId = existingDraft ? existingDraft.id : DraftManager.generateDraftId();
+            const draftData = {
+                title,
+                body,
+                createdAt: existingDraft ? existingDraft.createdAt : new Date().toISOString()
+            };
+            
+            try {
+                await DraftManager.saveDraft(draftId, draftData);
+                modal.remove();
+                
+                // Refresh drafts view if we're currently viewing drafts
+                if (window.location.hash.startsWith('#drafts')) {
```

Comment:
This doesn't appear to be taking. When I click the View All Drafts button, I see the `#drafts` hash appear, but there is no change in the UI.